test/shared_test/test_ortops_hfberttokenizer.cc

```diff
@@ -141,3 +141,104 @@ TEST(hf_bert_tokenizer_opertor, test_default) {
     TestInference(*ort_env, model_path.c_str(), inputs[i], outputs[i]);
   }
 }
+
+TEST(hf_bert_tokenizer_operator, test_optional_output) {
```

Comment:
nit: instead of cut-and-pasting so much code it would be nicer to refactor the test logic into a helper function that had a bool for producing/validating the token_type_ids output.